c7n/resources/route53.py

```diff
@@ -212,3 +214,107 @@ def process_resource_set(self, domains, keys):
             client.delete_tags_for_domain(
                 DomainName=d[self.id_key],
                 TagsToDelete=keys)
+
+
+@HostedZone.action_registry.register('enable-query-logging')
```

Comment:
given this supports enabling and disabling, following other naming patterns, we would typically just use `set-query-logging`